### PR TITLE
Handle more attribute kinds in LazyTensorOperation.

### PR DESCRIPTION
**Pull request by bgogul**

### CODE REVIEW

File: Sources/TensorFlow/Core/LazyTensorOperation.swift [hunk 1]
```diff
@@ -153,6 +153,9 @@ class LazyTensorOperation: TensorOperation {
         case ConstTensor(TFETensorHandle)
         case TensorDataTypeValue(TensorDataType)
         case TensorFunctionPointer(_TensorFunctionPointer)
+        case TensorDataTypeArray([TensorDataType])
```
Comment (rxwei) [inline Sources/TensorFlow/Core/LazyTensorOperation.swift]: Enum cases should use camel case. Didn't notice it earlier.

File: Sources/TensorFlow/Core/LazyTensorOperation.swift [hunk 2]
```diff
@@ -659,6 +662,12 @@ extension LazyTensorOperation.Attribute: CustomStringConvertible {
         case .ConstTensor(let v): return v.valueDescription
         case .TensorDataTypeValue(let v): return dataTypeAsString(v)
         case .TensorFunctionPointer(let v): return "TFFunction(\(v.name))"
+        case .TensorDataTypeArray(let values):
+            let descriptions = (values.map { dataTypeAsString($0) })
```
Comment (rxwei) [inline Sources/TensorFlow/Core/LazyTensorOperation.swift]:
```suggestion
            let descriptions = values.map { dataTypeAsString($0) }
```

File: Sources/TensorFlow/Core/LazyTensorOperation.swift [hunk 3]
```diff
@@ -659,6 +662,12 @@ extension LazyTensorOperation.Attribute: CustomStringConvertible {
...
+            let descString = descriptions.joined(separator: ", ")
+            return "[\(descString)]"
+        case .OptionalTensorShape(let t):  return String(describing: t)
```
Comment (rxwei) [inline Sources/TensorFlow/Core/LazyTensorOperation.swift]:
```suggestion
        case .OptionalTensorShape(let t): return String(describing: t)
```

File: Sources/TensorFlow/Core/LazyTensorOperation.swift [hunk 4]
```diff
@@ -659,6 +662,12 @@ extension LazyTensorOperation.Attribute: CustomStringConvertible {
...
+        case .OptionalTensorShapeArray(let t):  return "\(t)"
```
Comment (rxwei) [inline Sources/TensorFlow/Core/LazyTensorOperation.swift]:
```suggestion
        case .OptionalTensorShapeArray(let t): return "\(t)"
```

File: Sources/TensorFlow/Core/LazyTensorOperation.swift (hunk 1 above)
Comment (bgogul) [reply Sources/TensorFlow/Core/LazyTensorOperation.swift]: Will send a follow up PR for this.

**Pull request merged**